### PR TITLE
Created BottomSheet component

### PR DESCRIPTION
**Pull request by saxenanihal95**
PR for #1266 (Bottom Sheet Component)

Right Now i have added three props:

buttonProps: which will take all props of Button Component of React Native Elements

cancelButtonIndex : index of Button which will be the index of list of Bottom Sheet and will close on Click

list: Array of list item props of ListItem Component which will show in list of ButtomSheet component

### CODE REVIEW

**Comment by flyingcircle** [comment]
I'm not sure I understand the `buttonProps` field.  There's a cancel button that usually goes at the bottom, and I can understand that there could be a list of buttons, but `buttonProps` makes it sound like there's only one button. It might be good to have an example with pictures as well with the documentation.

**Comment by saxenanihal95** [comment]
@flyingcircle will add example with pictures 

and `buttonProps` is for the button which shows the ActionSheet Component

**Comment by saxenanihal95** [comment]
@flyingcircle updated PR with examples

**Comment by flyingcircle** [comment]
@saxenanihal95 We are ready to start finishing up this component and getting it into RNE. I recently simplified the react-native-elements-app project. So please add a tab for showing an example of how to use this.

File: src/bottomSheet/BottomSheet.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+const styles = StyleSheet.create({
+  safeAreaView: { flex: 1, backgroundColor: 'rgba(0,0,0,0.2)' },
+  modalView: {
+    flex: 1,
+    flexDirection: 'column-reverse',
+  },
+  listContainer: { backgroundColor: 'white' },
+});
+
```
Comment (flyingcircle) [inline src/bottomSheet/BottomSheet.js:85]: To follow the pattern, `BottomSheet.propTypes` needs to be defined here.

**Comment by airen29** [comment]
Could we maybe add a functionality that will close the BottomSheet when button is pressed? Not that the only last one?

For example:
```
onPressClose: () => {
              this.setVisible(false);
              onPress && onPress();
            },
```

and `onPressClose` will be by default set to `false`.

**Pull request merged**